Tests/TreePrinterTests/TreePrinterTests.swift

```diff
@@ -451,6 +451,20 @@ class TreePrinterTests: XCTestCase {
         XCTAssertTrue(lines[15].starts(with: options.finalConnector))
     }
     
+    func testStringInterpolation() {
+        let tree = TreeNode(title: "Root",
```

Comment:
There's a convenience method that would cut down the size of this test — you can use [`TreeNode.sampleTree`](https://github.com/cliss/TreePrinter/blob/8797ae14b14e1054f1f50d0197bd1121302d8273/Tests/TreePrinterTests/TreeNode.swift#L18) here instead.

```swift
let tree = TreeNode.sampleTree
```